rancilio-pid/userConfig.h

```diff
@@ -68,8 +71,10 @@
 #define MQTT_SERVER_PORT 1883    
 
 // BLynk
-#define AUTH "myauth"
```

Comment:
Also apologies for including these blynk details -can be safely ignored here too.